perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2519,5 +2525,27 @@ public void setPerunBl(PerunBl perunBl) {
 		this.perunBl = perunBl;
 	}
 
+	private String createUserAccountInExternalSystem (PerunSession session, User userToBeSponsored, String namespace, String attributeValue, String loginAttributeName, String password) {
```

Comment:
Please create a short JavaDoc for this method (You can find inspiration in other private methods).